<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseCfg</class>
 <widget class="QWidget" name="MouseCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="lstCmd">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Button</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCmd">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAlt">
       <property name="text">
        <string>Alt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkCtrl">
       <property name="text">
        <string>Ctrl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShift">
       <property name="text">
        <string>Shift</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
